config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_containerclusters.container.cnrm.cloud.google.com.yaml

```diff
@@ -532,10 +578,9 @@ spec:
               enableAutopilot:
                 description: Immutable. Enable Autopilot for this cluster.
                 type: boolean
-              enableBinaryAuthorization:
```

Comment:
Removing a field is a breaking change for a Config Connector CRD and right now we don't accept such breaking changes. This field is removed in Terraform because they had a major version bump, and this doesn't apply to Config Connector.

To unblock the in-place update of the taint, I suggest instead of copying over the entire go files, you copy the code logic only relevant to the taint feature to the Terraform library under third_party/ folder.